lib/feedsme/index.js

```diff
@@ -344,14 +393,23 @@ Feedsme.prototype._triggerStrategy = function _triggerStrategy({ releaseLine, ro
   // increment so it doesnt mess with future publishes
   //
   const previousMajor = semver.major(rootVersion) < semver.major(rootDepVersion);
-
-  const trigger = previousMajor ? true : inclusive;
+  const previousPublishedMajor = semver.major(rootVersion) === semver.major(latestRootPkgVersion);
```

Comment:
some comments here would be good. because this reads really weird. Explain how this test means that it's a previous published version.  From outside this PR it seems like this would be just saying, "is the current root version the latest published version to npm" when wouldn't this comparison here be true?

Reply:
The names for these variables need an overhaul – agreed on that point. In in this case the "root" name is confusing since it is somewhat interchangeable with "parent" but it's unclear when "pkg" is a dependent package or not.

That said, I think we should simply open an issue outlining this confusion and then do a pass after this ships. Objections?

Reply:
I think we should add a comment about what we're doing here now, and open an issue to come back and revisit the function and clean it up along with addressing the new comment about no longer needing to walk the release lines since we have the full published version set now.

Reply:
Commented but yea this code is confusing :(